pkg/cloud/aws/aws.go

```diff
@@ -1,3 +1,5 @@
+//go:build !non_deploy
```

Comment:
No need to add constraints in this file. Applies to all files under pkg/. Only files under cmd/subctl should have the constraint.